sonar-helm-for-iac/helm-imports.go

```diff
@@ -24,6 +24,7 @@ import (
 	"errors"
 	"github.com/BurntSushi/toml"
 	"github.com/Masterminds/sprig/v3"
+	"github.com/SonarSource/sonar-iac/sonar-helm-for-iac/converters"
```

Comment:
Why is this `github.com/...` necessary` and we can simply import `converters`? Is it the fqn like in java?

Reply:
Yes, like FQN, which is the same both when you include this module in the same project or in another